jobs/ads-incrementality-dap-collector/README.md

```diff
@@ -0,0 +1,142 @@
+# Ads Incrementality DAP collector
+
+## Background
+
+Incrementality is a way to measure the effectiveness of our ads in a general, agreggated, privacy-preserving way --
+without knowing anything about specific users.
+
+Incrementality works by dividing clients into various Nimbus experiment branches that vary how/whether an ad is shown.
+Separately, a [DAP](https://docs.divviup.org/) task is configured to store the metrics for each experiment branch in a
+different DAP bucket.
+
+Firefox is instrumented with [DAP telemetry functionality](https://github.com/mozilla-firefox/firefox/tree/main/toolkit/components/telemetry/dap), which allows it to send metrics and reports into the correct DAP buckets as configured in the experiment.
+
+Then this job can go out and collect metrics from DAP (using bucket info from the experiment's data), and write them
+to BQ.
+
+Some examples of existing metrics are
+- "url visit counting", which increments counters in DAP when a firefox client visits an ad landing page.
+
+Great care is taken to preserve privacy and anonymity of these metrics. The DAP technology itself agreggates counts
+in separate systems and adds noise. The DAP telemetry feature will only submit a count to DAP once per week per client.
+All DAP reports are deleted after 2 weeks.
```

Comment:
Details of DAP usage are not needed either since they are not relevant to this ETL job.